.pre-commit-config.yaml

```diff
@@ -29,9 +29,7 @@ repos:
       - id: isort
         args: [--settings-path=pyproject.toml]
 
-  # - repo: https://github.com/pre-commit/mirrors-mypy
-  #   rev: v0.790
-  #   hooks:
-  #     - id: mypy
-  #       language: system
-  #       args: [--config-file=mypy.ini]
+  - repo: https://github.com/RobertCraigie/pyright-python
+    rev: v1.1.284
```

Comment:
the newest version is 1.1.287, maybe try to run a test and see if it works out of the box for us

Reply:
I will check for the newest version, but Microsoft is releasing minor releases few times a month so it will be hard to always update it